src/kernel/disconnected_transactions.h

```diff
@@ -12,7 +12,10 @@
 
 #include <list>
 #include <unordered_map>
+#include <vector>
 
+/** Maximum kilobytes for transactions to store for processing during reorg */
+static const unsigned int MAX_DISCONNECTED_TX_POOL_SIZE = 20'000;
```

Comment:
Nit: It is just moved here, but everwhere this is used we multiply by `1000`, so why define it in kilobytes in the first place?

Reply:
Also, if you do this, how about replacing _SIZE  with _BYTES ?

Reply:
Agree the fact that it's kb should be documented / baked in. Going to say out of scope for this PR. Should be easy to do in a followup.